.circleci/config.yml

```diff
@@ -335,10 +334,63 @@ jobs:
             https://api.github.com/repos/NCAR/VAPOR/issues/${CIRCLE_PULL_REQUEST##*/} \
             -d "$json"
 
-  build_ubuntu18:
+  python_api_centos:
+    docker:
+      - image: sgpearse/vapor3-centos7:latest
+
+    resource_class: large
+
+    steps:
+      - run:
+          name: update 3rd party libraries
+          command: |
+            rm -rf /usr/local/VAPOR-Deps/2019-Aug
+            pip3 install gdown --upgrade
+            gdown https://drive.google.com/uc?id=1S9DwySMnQrBuUUZGKolD__WQrjTmLgyn
+            filename="/root/project/2019-Aug-CentOS.tar.xz"
+            bsdtar -xf ${filename} -C /usr/local/VAPOR-Deps
+
+      - run:
+          name: install python
+          command: |
+            yum -y update || true
+            yum install -y python3
+            yum install -y python3-pip
+            pip3 install --upgrade pip
+            pip3 install scikit-build command
+
+      - run:
+          name: install miniconda
+          command: |
+            yum install -y wget
+            wget https://repo.continuum.io/miniconda/Miniconda3-latest-Linux-x86_64.sh -O ~/miniconda.sh
+            bash ~/miniconda.sh -b -p ~/miniconda 
+
+      - checkout
+
+      - run:
+          name: conda build .
+          command: |
+            yum -y install doxygen
+            rm -rf /root/project/build/*
+            rm -f /root/project/site.local
+            cd /root/project/build
+            cd /root/project/conda
+            export PYTHONPATH=/root/miniconda/lib/python3.9
+            /root/miniconda/bin/conda install -y conda-build
+            /root/miniconda/bin/conda config --add channels conda-forge
+            /root/miniconda/bin/conda build .
+          no_output_timeout: 20m
+ 
+      - store_artifacts:
+          path: /root/miniconda/conda-bld/linux-64
```

Comment:
We only need the tarball.

Reply:
CircleCI is finicky about how it uploads artifacts, in that wildcards can't be used to select files.  Only directories can be targeted.  I'll just move the tarball to an empty directory with the wildcard, and then target that directory.